content/en/docs/gke/deploy/deploy-cli.md

```diff
@@ -180,6 +180,13 @@ To deploy kubeflow, run the following command:
 
      * This issue is being tracked in [kubeflow/manifests#1234](https://github.com/kubeflow/manifests/issues/1234)
 
+   * If resources can't be created with error message like:
+    ```
```

Comment:
This isn't rendering correctly... Please add a line break between the list item and the code block.